TLC5957.h

```diff
@@ -0,0 +1,67 @@
+#ifndef TLC5957_H
+#define TLC5957_H
+
+#include <stdint.h>
+#include <SPI.h>
+
+#define GS_BITS 16
```

Comment:
I think it was a mistake to define these in the header of the TLC5955 and TLC5947.

If one defines values like this, they should typically be prefixed with something much more unique.

```
#define TLC5957_GS_BITS 16
```
is more appropriate.

However, since they are only meant to be used by the `TLC5957.cpp` file, it is likely best to define them in there, and to avoid naming collisions.